### ISSUE TITLE
warn on mixing ?? with > or ==

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1233]
```javascript
config?.items?.length ?? 0 > 0
// intended: (config?.items?.length ?? 0) > 0
// actual: config?.items?.length ?? (0 > 0)
```

https://x.com/strager/status/1876508636236964158

**Comment by VIJESHG** [comment]
Hello @strager I have created PR for this issue: https://github.com/quick-lint/quick-lint-js/pull/1234 Please review and add
